data.sql

```diff
@@ -3,4 +3,11 @@
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03', 10.23, true, 0);
```

Comment:
Great job @yonas44 🤝

- [ ] As specified in the project requirements, kindly maintain all negative values and type as float or decimal for all `weight_kg` columns.

Reply:
Okay, thank you for all your feedback🙏

Reply:
* This issue pointed out by the previous CR has not been fixed as the `weight_kg` column contains some negative( - ) values below 👇 is a screenshot of these values 
![image](https://user-images.githubusercontent.com/52497246/212961670-6ba95ac8-9622-464c-9e3d-c8f773e80c64.png)